app/build.gradle

```diff
@@ -80,13 +80,16 @@ android {
 
 dependencies {
 // AndroidX
+    implementation "androidx.multidex:multidex:2.0.1"
     implementation 'androidx.appcompat:appcompat:1.3.1'
     implementation 'androidx.cardview:cardview:1.0.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.1'
     implementation 'androidx.exifinterface:exifinterface:1.3.3'
     implementation 'androidx.preference:preference:1.1.1'
     implementation 'com.google.android.material:material:1.4.0'
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
+// Splash Screen
+    implementation 'androidx.core:core-splashscreen:1.0.0-alpha01'
```

Comment:
"A newer version of androidx.core:core-splashscreen than 1.0.0-alpha01 is available: 1.0.0-alpha02"

Should probably use the new version :)